src/main/common/axis.h

```diff
@@ -42,6 +42,7 @@ typedef enum {
     AI_PITCH
 } angle_index_t;
 
-#define ANGLE_INDEX_COUNT 2
+#define RP_AXIS_COUNT 2
+#define EF_AXIS_COUNT 2
```

Comment:
But that means that body angles will be mixed with earth frame distances. It will be even more confusing that it is now ... 
And earth frame has usually 3 dimensions, using 2d for flying thing is a bit strange.

And all this comes from need to name number 2 ...

Reply:
Yeah agreed. What about `XY_AXIS_COUNT` instead of "EF_AXIS_COUNT" or something similar?

Reply:
Problem is that when checking the code, XY doesn't tell us whether the values are earth frame axis count or body frame axis count.
In this case the axis counts are always 2, so the whole discussion is pointless in my opinion, but at least using a descriptive name where the counter is looping through an earth reference set of axes vs a body reference set of axes helps make the code more readable, I think.  I only added the names to make it easier to be sure that the loop count was correct for the purpose of the loop.

Reply:
> XY doesn't tell us whether the values are earth frame axis count or body frame axis count.

there are other way to express it, like variable suffix or comment. This macro is mainly related to problem size, 2d on this case

Reply:
Must say I liked the simplicity of [2] :-)

Reply:
Anyway I think we are using arraylen.